src/spec/test/ClassDesignASTTransformsTest.groovy

```diff
@@ -189,14 +189,14 @@ class NumberBooleanBuilder {
     StringBuilder nums = new StringBuilder()
     @Delegate(includeTypes=[AppendFloatSelector], interfaces=false)
     StringBuilder bools = new StringBuilder()
-    String result() { "${nums.toString()} | ${bools.toString()}" }
+    String result() { "${nums.toString()} ~ ${bools.toString()}" }
```

Comment:
This had to be changed because using a pipe here broke the formatting of asciidoc table this snippet was used in.